Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -152,6 +152,24 @@
 
       do_blocking_move_to_xy(destination);
       homeaxis(Z_AXIS);
+      #if LINEAR_AXIS >= 4
+        homeaxis(I_AXIS);
+      #endif
+      #if LINEAR_AXIS >= 5
+        homeaxis(J_AXIS);
+      #endif
+      #if LINEAR_AXIS >= 6
+        homeaxis(K_AXIS);
+      #endif
+      #if LINEAR_AXIS >= 7
+        homeaxis(U_AXIS);
+      #endif
+      #if LINEAR_AXIS >= 8
+        homeaxis(V_AXIS);
+      #endif
+      #if LINEAR_AXIS >= 9
+        homeaxis(W_AXIS);
+      #endif
```

Comment:
I'm not sure how extra axes ought to be applied in the safe homing of Z. However, it is truly just meant for Z and there will have to be some other options pertaining to whether extra axes should require XY homing as part of their homing.

Reply:
I will revert this change, I had a stupid missconception of the code.

Reply:
done